spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -334,7 +334,10 @@ class AbstractSpinnakerBase(ConfigHandler, SimulatorInterface):
         "_vertices_or_edges_added",
 
         # Set of all seen vertext labels
-        "_vertext_labels"
+        "_vertext_labels",
```

Comment:
This should be "vertex", not "vertext"; but it also doesn't seem to be used anywhere either.

Reply:
Ah yes, unfortunately my only change is to add a comma, so this was already in there.  I think it gets used by classes in other modules, so changing it in this PR is out-of-scope and in general is more of a pain...

Reply:
ill just go with oops. as i persume thats me.

Reply:
I did a search for it elsewhere and didn't find it, which is why I brought it up here.

Reply:
True enough - it is now gone